### PR TITLE
Possibly fix the Onesync Spectate Issue Mentioned in Issue #137

### PR DESCRIPTION
**Pull request by amantu-qbit**
Added Requested Collision Before the TP. Should Stick the Spectate with eh person which nullifies the problem we face with one sync. And it might fix the issue mentioned in #137 . I don't have a player base to test. Would be nice if anyone could try and give me feedback.

### CODE REVIEW

**Comment by sinphoenix** [comment]
I tested with one person. I will do another test today with more people to ensure it works.

**Comment by kokorog** [comment]
dont work for me

**Comment by GodyFromDiscord** [comment]
PR Outdated from main, will test soon.

**Comment by amantu-qbit** [comment]
> Thank you, will test shortly and feedback to the team

Thank you. Did some changes without being tested would greatly appreciate it if can get feedback!

File: locales/en.lua [hunk 1]
```diff
@@ -15,6 +15,7 @@ local Translations = {
         ["failed_set_speed"] = "You did not set a speed.. (`fast` for super-run, `normal` for normal)",
         ["failed_set_model"] = "You did not set a model..",
         ["failed_entity_copy"] = "No freeaim entity info to copy to clipboard!",
+        ["no_player"] = 'Player Doesn\'t exist in server anymore',
```
Comment (GodyFromDiscord) [inline locales/en.lua:18]: Just reviewing and testing in game at the moment, but for the sake of grammar, if you don't make making 'Doesn\'t' to 'doesn\'t'

File: client/client.lua [hunk 2]
```diff
@@ -1141,6 +1148,10 @@ RegisterNetEvent('qb-admin:client:ToggleCoords', function()
     ToggleShowCoordinates()
 end)
 
+menu7_dev_copy_vec2:On("select", function()
+    CopyToClipboard('coords2')
```
Comment (GodyFromDiscord) [inline client/client.lua:1152]: This all works fine and looks good to me

**Review by GodyFromDiscord — CHANGES_REQUESTED** [review]
Just testing the spectate function now but I have also requested a tiny grammar change

**Comment by GodyFromDiscord** [comment]
Upon spectating the player you contiously fall through the ground and it does not spectate them, this happens at a distance only, but when within a close range of the player, the spectate works fine

**Comment by amantu-qbit** [comment]
> Upon spectating the player you continuously fall through the ground and it does not spectate them, this happens at a distance only, but when within a close range of the player, the spectate works fine

Thank you for the info. May I know apprx distance?

**Comment by GodyFromDiscord** [comment]
> > Upon spectating the player you continuously fall through the ground and it does not spectate them, this happens at a distance only, but when within a close range of the player, the spectate works fine
> 
> Thank you for the info. May I know apprx distance?

I will create a video, one moment

**Comment by GodyFromDiscord** [comment]
https://medal.tv/games/gta-v/clips/17WykA4Rlw1_pH/d1337SMnVVKv?invite=cr-MSw3WTMsNjI2NTgyMSw

Video ^

**Comment by amantu-qbit** [comment]
> https://medal.tv/games/gta-v/clips/17WykA4Rlw1_pH/d1337SMnVVKv?invite=cr-MSw3WTMsNjI2NTgyMSw
> 
> Video ^

Thanks for the vid. Appreciate it!

meanwhile Did an update can you try?

**Comment by GodyFromDiscord** [comment]
> > https://medal.tv/games/gta-v/clips/17WykA4Rlw1_pH/d1337SMnVVKv?invite=cr-MSw3WTMsNjI2NTgyMSw
> > Video ^
> 
> Thanks for the vid. Appreciate it!
> 
> meanwhile Did an update can you try?

Still facing the same issue as before with the latest push made

**Pull request closed without merging**